libs/utils/decorate.js

```diff
@@ -0,0 +1,146 @@
+//
+// Shared decorate utils for Consonant blocks
+// Block usage ref: [media, z-pattern]
```

Comment:
probably don't need this comment - good chance it won't be updated even if other blocks use decorate.js